.github/workflows/universalnrp-test.yml

```diff
@@ -11,7 +14,18 @@ jobs:
       matrix:
         target:
```

Comment:
> target: [](http://example.com/codeflow?start=7&length=8)

Where we differentiate the targets among the set with OSConfig vs the set without OSConfig?

Reply:
We need both sets run on separate VMs every run.

Reply:
All of these sets are without osconfig, should i also include all these targets with it installed as well?

Reply:
Yes, please. We need the set duplicated with equal number of distros: with and without the OSConfig package installed.